frontend/src/store/configurate-podcast/actions.ts

```diff
@@ -22,7 +22,7 @@ const create = createAsyncThunk<void, PodcastFormPayload, AsyncThunkConfig>(
     const podcast = await podcastApi.create({
       userId: (<User>auth.user).id,
       description: podcastPayload.description,
-      genreId: Number(podcastPayload.genre),
+      genreId: (podcastPayload.genre === null) ? null : Number(podcastPayload.genre),
```

Comment:
```suggestion
      genreId: podcastPayload.genre ?? Number(podcastPayload.genre),
```

Reply:
> 

This way we cannot to send null to the server.

Reply:
genreId: number | null

Reply:
got you